bokehjs/test/integration/widgets.ts

```diff
@@ -205,6 +205,11 @@ describe("Widgets", () => {
     await view.ready
   })
 
+  it.allowing(8)("should allow AutocompleteInput with search_strategy='includes'", async () => {
+    const obj = new AutocompleteInput({placeholder: "Enter value ...", search_strategy: "includes"})
+    await display(obj, [500, 100])
```

Comment:
This test doesn't do anything useful for the expense it brings. A test like this should result in some difference either in the resulting `*.blf` or `*.png` files, or both, when compared to a basic integration test (two entries above) for the same widget. Or it should at least contain some assertions. I would move here the test code you wrote for the defunct integration tests in Python, and adjust it, so that it doesn't use CSS selectors if not strictly needed, etc.

Reply:
👍 thanks for the feedback. I've addressed your other two comments but moving the test over to js will take me a little longer so I'll have a go over the weekend. Would you like the Python test I added to be removed?

Reply:
Here's where I've got to. I'm fairly sure the part from `const menu ...` will work, but I don't think I am simulating a user entering input correctly beforehand. Any ideas @mattpap? Thank you.

```
    const completions = ["123", "1123", "231", "1324", "3211"]
    const obj = new AutocompleteInput({placeholder: "Enter value ...", completions, search_strategy: "includes"})
    const {view} = await display(obj, [500, 300])
    obj.value = "23"
    view.change_input()
    await view.ready                                                                               // not sure if necessary
    
    assert(view.model.value === "23")                                                // this passes
    const menu = view.shadow_el.querySelector(".bk-menu")
    const matchedCompletions = menu!.querySelectorAll("div")
    assert(matchedCompletions.length === 3)                                   // fails here
    assert(matchedCompletions[0]!.getAttribute("text") === "123")
    assert(matchedCompletions[1]!.getAttribute("text") === "1123")
    assert(matchedCompletions[2]!.getAttribute("text") === "231")
```

Reply:
I figured it out, needed to add a focus event to make the completions appear 🎉